src/Illuminate/Foundation/Testing/TestResponse.php

```diff
@@ -136,6 +136,20 @@ public function assertUnauthorized()
         return $this;
     }
 
+    /**
+     * Assert that the response has the given status code and no content.
+     *
+     * @param  int  $status
+     * @return $this
+     */
+    public function assertNoContent($status = 204)
+    {
+        $this->assertStatus($status);
+        PHPUnit::assertEmpty($this->getContent(), 'Response content is not empty.');
```

Comment:
Add a whiteline above the PHPUnit call here:

```suggestion

        PHPUnit::assertEmpty($this->getContent(), 'Response content is not empty.');
```

Reply:
It's gotta breathe... 🤣